libc/src/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ add_subdirectory(stdio)
 add_subdirectory(stdlib)
 add_subdirectory(string)
 add_subdirectory(wchar)
+add_subdirectory(dlfcn)
```

Comment:
Look like this will have to go to the other PR.

Reply:
🎉